From e78af5ffe53a0d24854d721d1166a60f8ed0dfb6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Thu, 4 Sep 2014 12:30:46 -0400
Subject: [PATCH] localed: rename write_data_x11 to x11_write_data

Other functions in this file follow this pattern,
we have vconsole_write_data and locale_write_data.
---
 src/locale/localed.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/locale/localed.c b/src/locale/localed.c
index 2252080f2e..1d5be715ba 100644
--- a/src/locale/localed.c
+++ b/src/locale/localed.c
@@ -430,7 +430,7 @@ static int vconsole_write_data(Context *c) {
         return write_env_file_label("/etc/vconsole.conf", l);
 }
 
-static int write_data_x11(Context *c) {
+static int x11_write_data(Context *c) {
         _cleanup_fclose_ FILE *f = NULL;
         _cleanup_free_ char *temp_path = NULL;
         int r;
@@ -605,7 +605,7 @@ static int vconsole_convert_to_x11(Context *c, sd_bus *bus) {
         if (modified) {
                 int r;
 
-                r = write_data_x11(c);
+                r = x11_write_data(c);
                 if (r < 0) {
                         log_error("Failed to set X11 keyboard layout: %s", strerror(-r));
                         return r;
@@ -1050,7 +1050,7 @@ static int method_set_x11_keyboard(sd_bus *bus, sd_bus_message *m, void *userdat
                     free_and_strdup(&c->x11_options, options) < 0)
                         return -ENOMEM;
 
-                r = write_data_x11(c);
+                r = x11_write_data(c);
                 if (r < 0) {
                         log_error("Failed to set X11 keyboard layout: %s", strerror(-r));
                         return sd_bus_error_set_errnof(error, r, "Failed to set X11 keyboard layout: %s", strerror(-r));
